pkg/controller/compliancescan/scan.go

```diff
@@ -508,6 +436,7 @@ func (r *ReconcileComplianceScan) newPlatformScanPod(scanInstance *compv1alpha1.
 						"--tls-client-cert=/etc/pki/tls/tls.crt",
 						"--tls-client-key=/etc/pki/tls/tls.key",
 						"--tls-ca=/etc/pki/tls/ca.crt",
+						"--disable-raw-upload=" + getDisableRawResultUploadValue(scanInstance),
```

Comment:
This could mirror the other comment - where we saw something like `--upload-raw-results=true|false`